plugin-start

```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
```

Comment:
@wdurairaj do we need validate this file again ? looks like moved to different folder..